submissions/AndriyBerezovskiy/hooli-style-popup/index.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>popup</title>
+    <link rel="icon" type="image/x-icon" href="./img/favicon.svg" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header__menu">
+        <ul class="header__list df">
+          <li>
+            <a class="header__link" href="#">Mail</a>
+          </li>
+          <li>
+            <a class="header__link" href="#">Images</a>
+          </li>
+          <li class="popup">
+            <input type="checkbox" id="popup__input" class="popup__input" />
+            <label for="popup__input" class="popup__icon">
+              <img src="./img/menu.svg" alt="menu" />
+            </label>
+            <div class="popup__content">
+              <div class="popup__wrap">
+                <ul class="popup__list">
+                  <li class="popup__item df">
+                    <a class="popup__link df" href="#">
+                      <img
+                        class="popup__img"
+                        src="./img/location.svg"
+                        alt="location"
```

Comment:
I read this quite informative and new article for me. And in the process of refactoring my `alt`, I deleted some altogether, because it is written there that if the icon is in the button, then the `alt` is not needed. And when the drop-down menu is opened, we have exactly the same situation, if I'm not mistaken. I have described some `alt` in more detail according to what I have read.